Description: fix spelling of 'separate' in the man page and source code.
Author: Miah Gregory <mace@debian.org>
Last-Update: 2012-05-12
Index: dcfldd-1.3.4.1/dcfldd.c
===================================================================
--- dcfldd-1.3.4.1.orig/dcfldd.c
+++ dcfldd-1.3.4.1/dcfldd.c
@@ -181,7 +181,7 @@ Copy a file, converting and formatting a
                              in a comma separated list\n\
   hashlog=FILE             send MD5 hash output to FILE instead of stderr\n\
                              if you are using multiple hash algorithms you\n\
-                             can send each to a seperate file using the\n\
+                             can send each to a separate file using the\n\
                              convention ALGORITHMlog=FILE, for example\n\
                              md5log=FILE1, sha1log=FILE2, etc.\n\
   hashlog:=COMMAND         exec and write hashlog to process COMMAND\n\
